processor/k8sattributesprocessor/internal/kube/client.go

```diff
@@ -846,6 +859,19 @@ func (c *WatchClient) addOrUpdateReplicaSet(replicaset *apps_v1.ReplicaSet) {
 	c.m.Unlock()
 }
 
+// This function removes all data from the Pod except what is required by extraction rules
```

Comment:
```suggestion
// This function removes all data from the ReplicaSet except what is required by extraction rules
```
I assume this should be ReplicaSet? I might be missing something though.

Reply:
Thanks for spotting this, it's just bad copy paste.